### PR TITLE
Dvernier signoff

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,189 +1,7 @@
-# multicloud-operators-subscription-release
+<p align="center"><a href="http://35.227.205.240/?job=build_go-repo-template_postsubmit">
+<!-- prow build badge, godoc, and go report card-->
+<img alt="Build Status" src="http://35.227.205.240/badge.svg?jobs=build_go-repo-template_postsubmit">
+</a> <a href="https://godoc.org/github.com/IBM/go-repo-template"><img src="https://godoc.org/github.com/IBM/go-repo-template?status.svg"></a> <a href="https://goreportcard.com/report/github.com/IBM/go-repo-template"><img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/IBM/go-repo-template" /></a> <a href="https://codecov.io/github/IBM/go-repo-template?branch=master"><img alt="Code Coverage" src="https://codecov.io/gh/IBM/go-repo-template/branch/master/graphs/badge.svg?branch=master" /></a></p>
```
Comment (kuanf) [inline README.md]: please replace the `go-repo-temlate` with your repo name  `multicloud-operators-subscription-release`

Comment (itdove) [reply README.md]: Done

**Pull request merged**